core/src/components/range/range.md.scss

```diff
@@ -63,6 +64,32 @@
   transition-timing-function: ease;
 
   z-index: 2;
+
+  &::before {
+    @include border-radius(50%);
+    @include position(null, null, null, 0);
+
+    @include rtl() {
+      /* stylelint-disable-next-line property-disallowed-list */
+      left: unset;
+    }
```

Comment:
Why is the RTL mixin needed?

Reply:
These are outdated styles from previous changes 👍 removed them & confirmed they do not impact the display of the indicator.